mujoco_warp/viewer.py

```diff
@@ -49,10 +49,6 @@ def key_callback(key: int) -> None:
 
 def _load_model():
```

Comment:
do we anticipate we will need this again in the future?

if not, shall we revert to just calling `MjModel.from_xml_path`?